pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+#Creating eve cgroup which will be parent cgroup for all eve services
+mkdir -p /sys/fs/cgroup/cpuset/eve
+mkdir -p /sys/fs/cgroup/cpu/eve
+mkdir -p /sys/fs/cgroup/cpuacct/eve
+mkdir -p /sys/fs/cgroup/blkio/eve
+mkdir -p /sys/fs/cgroup/memory/eve
+mkdir -p /sys/fs/cgroup/devices/eve
+mkdir -p /sys/fs/cgroup/freezer/eve
+mkdir -p /sys/fs/cgroup/net_cls/eve
+mkdir -p /sys/fs/cgroup/perf_event/eve
+mkdir -p /sys/fs/cgroup/net_prio/eve
+mkdir -p /sys/fs/cgroup/hugetlb/eve
+mkdir -p /sys/fs/cgroup/pids/eve
+mkdir -p /sys/fs/cgroup/systemd/eve
```

Comment:
This is almost begging for someone to make a mistake. Can we put this in a for loop?

```sh
CGROUPS="cpuset cpu cpuacct blkio memory freezer " # etc etc
for cg in $CGROUPS; do
  mkdir -p /sys/fs/cgroup/${cg}/eve
done
```

Reply:
Changed.